File: public/phase-2-20160620022203/C/magic-square.c
// Name : Magic Square
#include <stdio.h>
int main() {
	// input:
	// original_magic_square : original magic square
	int original_magic_square[3][3], repaired_magic_square[3][3]={{4,4,4},{4,4,4},{4,4,4}}, idx, jdx;
	char original_magic_square_tmp[3], repaired_magic_square_tmp[3];
	for(idx = 0; idx< 3; idx++) {
		scanf("%s", &original_magic_square_tmp[0]);
		for(jdx = 0;jdx<3;jdx++) {
			original_magic_square[idx][jdx] = original_magic_square_tmp[jdx];
		}
	}


	// write your code here
	// store your results in `repaired_magic_square`

	// output
	for(idx = 0; idx< 3; idx++) {
		for(jdx = 0;jdx<3;jdx++) {
			printf("%d ",repaired_magic_square[idx][jdx]);
		}
		printf("\n");
	}
	return 0;
}

File: public/phase-2-20160620022203/PYTHON/magic-square.py
#!/usr/bin/python
# Name : Magic Square
# input:
# original_magic_square : original magic square
original_magic_square = []
for _ in xrange(3):
	original_magic_square_elements = raw_input().strip().split(' ')
	original_magic_square.append(map(int, original_magic_square_elements))


# write your code here
# store your results in `repaired_magic_square`

# output

# Dummy Data
repaired_magic_square = [[4,4,4],[4,4,4],[4,4,4]]

for idx in xrange(3):
	print(" ".join(str(e) for e in repaired_magic_square[idx]))